packages/svelte/tests/validator/samples/static-state-reference/input.svelte

```diff
@@ -6,11 +6,6 @@
 	console.log(obj);
 	console.log(count);
 	console.log(doubled);
-	// these are ok because they're writes
-	count++;
-	count = 1;
-	obj.a++;
-	obj.a = 1;
```

Comment:
?

Reply:
oh i did a `git checkout dash-case-warnings -- test/validator` to bring the tests over from the other PR